pkg/test/masteruserrecord/assertion.go

```diff
@@ -73,6 +73,15 @@ func (a *Assertion) AllUserAccountsHaveStatusSyncIndex(syncIndex string) *Assert
 	return a
 }
 
+func (a *Assertion) AllUserAccountsHaveCluster(expected toolchainv1alpha1.Cluster) *Assertion {
```

Comment:
Does it make sense that all `UserAccounts` in one MUR have the same cluster? it's not really realistic case - `UserAccounts` should be provisioned in different clusters...

Reply:
You are right. It won't be realistic when we support multiple UserAccounts. So that assertion won't be useful for checking MURs with multiple UserAccounts. But we are not going to support multiple accounts any time soon. And for now it's still useful to have such an assertion.

I wanted to provide something quick and simple for the current usecases and we can introduce another more useful assertion for multiple accounts when we need it.